tests/test_infer_ops/triton/utils.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import math
+
+import torch
+from torch.nn import functional as F
+
+
+def benchmark(func, *args):
```

Comment:
Benchmark should not be in `tests` folder.

Reply:
removed.